data/json/items/gun/500.json

```
@@ -6,8 +6,9 @@
     "type": "GUN",
     "name": { "str_sp": "Big Horn Model 89" },
     "description": "Built in the image of the venerable Winchester Model 1886, Big Horn Armory's Model 89 rifle packs all the power of the .500 S&W Magnum cartridge in to a package less likely to break one's wrist.",
-    "weight": "3447 g",
+    "weight": "1995 g",
     "volume": "3 L",
```

```suggestion
    "weight": "3742 g",
    "volume": "1995 ml",
```
Swapped the field to change, whoops :3
(The weight on Big Horn's website is for the 16" model, I got this from the listing I used for volume+length calculations)